cfme/tests/services/test_rest_services.py

```diff
@@ -79,7 +79,11 @@ def is_retired(entity):
     # wait until retired, if this fails, settle with "retiring" state
     retval = wait_for(lambda: is_retired(entity), num_sec=num_sec, delay=10, silent_failure=True)
     if not retval:
-        assert entity.retirement_state == 'retiring'
+        try:
```

Comment:
I think I'd rather see this as `getattr(entity, 'retirement_state', None) == 'retiring'`, which accomplishes the same thing.